cranelift/filetests/filetests/alias/simple-alias.clif

```diff
@@ -11,62 +15,27 @@ function %f0(i64 vmctx, i32) -> i32, i32, i32, i32 {
 block0(v0: i64, v1: i32):
     v2 = heap_addr.i64 heap0, v1, 0
     v3 = load.i32 v2+8
+    ;; This should reuse the load above.
     v4 = heap_addr.i64 heap0, v1, 0
     v5 = load.i32 v4+8
+    ; check: v5 -> v3
+    
     call fn0(v0)
+
+    ;; The second load is redundant wrt the first, but the call above
+    ;; is a barrier that prevents reusing v3 or v5.
```

Comment:
Can we also have tests for the other fence-semantics instructions too? Just tiny functions for each of them like

```
v1 = load.i32 v0
<fence instruction>
v2 = load.i32 v0
;; check: ...
```

Reply:
Done! Added `filetests/alias/fence.clif`.